.env.sample

```diff
@@ -91,11 +99,11 @@ GH_HOST=https://github.com
 
 ## The name of the GitHub bot account the students will call
 ## You must have access to this account because it needs to be
-## added to both the admin and staff teams so it can admin and
+## added to the admin team so it can admin and
 ## comment on repos. Do not include the @ in the username.
 ## The bot needs to be added to your org with admin privileges
 ## e.g., for public GitHub here: https://github.com/orgs/ORGNAME/people
-GH_BOT_USERNAME=ubcbot
+GH_BOT_USERNAME=classytest-bot01
```

Comment:
Does this make sense in the sample file? I figured the sample would be for sample deploys (which will all use `ubcbot`, rather than sample test instances? But if that's not the intent this doesn't matter...

Reply:
Hmn... It really should be autobot.